arrows/matlab/mxarray.h

```diff
@@ -831,247 +773,208 @@ class MxArray
   inline bool isSparse() const { return mxIsSparse( m_array ); }
 
-  /** Determine whether input is structure array.
-   */
+  /// Determine whether input is structure array.
   inline bool isStruct() const { return mxIsStruct( m_array ); }
 
-  /** Determine whether array represents data as unsigned 8-bit integers.
-   */
+  /// Determine whether array represents data as unsigned 8-bit integers.
   inline bool isUint8() const { return mxIsUint8( m_array ); }
 
-  /** Determine whether array represents data as unsigned 16-bit integers.
-   */
+  /// Determine whether array represents data as unsigned 16-bit integers.
   inline bool isUint16() const { return mxIsUint16( m_array ); }
 
-  /** Determine whether array represents data as unsigned 32-bit integers.
-   */
+  /// Determine whether array represents data as unsigned 32-bit integers.
   inline bool isUint32() const { return mxIsUint32( m_array ); }
 
-  /** Determine whether array represents data as unsigned 64-bit integers.
-   */
+  /// Determine whether array represents data as unsigned 64-bit integers.
   inline bool isUint64() const { return mxIsUint64( m_array ); }
 
-  /** Determine whether a struct array has a specified field.
-   */
+  /// Determine whether a struct array has a specified field.
   bool hasField( const std::string& field_name, mwIndex index = 0 ) const
   {
     return isStruct() &&
            mxGetField( m_array, index, field_name.c_str() ) != NULL;
   }
 
 
-  /** Element size.
-   */
+  /// Element size.
   int elementSize() const { return mxGetElementSize( m_array ); }
 
-  /** Determine whether input is NaN (Not-a-Number).
-   */
+  /// Determine whether input is NaN (Not-a-Number).
   static inline bool IsNaN( double value ) { return mxIsNaN( value ); }
 
-  /** Value of infinity.
-   */
+  /// Value of infinity.
   static inline double Inf() { return mxGetInf(); }
 
-  /** Value of NaN (Not-a-Number).
-   */
+  /// Value of NaN (Not-a-Number).
   static inline double NaN() { return mxGetNaN(); }
 
-  /** Value of EPS.
-   */
+  /// Value of EPS.
   static inline double Eps() { return mxGetEps(); }
 
 
 private:
-  /** Copy constructor is prohibited except internally.
-   */
+  /// Copy constructor is prohibited except internally.
   MxArray( const MxArray& array );
 
   // MxArray(const MxArray& array) = delete;
-  /** Copy assignment operator is prohibited.
-   */
+  /// Copy assignment operator is prohibited.
   MxArray& operator=( const MxArray& rhs );
   // MxArray& operator=(const MxArray& rhs) = delete;
 
-  /*************************************************************/
-  /**             Templated mxArray importers                 **/
-  /*************************************************************/
+  //*************************************************************
+  ///
+  ///***           Templated mxArray importers                ***
+  ///
+  //*************************************************************
```

Comment:
This needs some manual massaging, and it doesn't look like it's *supposed* to be a Doxygen comment.

I'm not sure you want to be making this sort of change, but this looks like a candidate for a region marker (`//BEGIN` / `//END`).